src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -369,6 +370,8 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
 
                 // C# preview features.
                 //return LanguageVersion.Preview;
+                case MessageID.IDS_Missing:
+                    return LanguageVersion.Preview;
```

Comment:
I don't think we need an entry for `IDS_Missing` in the method since we're not treating `IDS_Missing` as a feature.